pixl_imaging/src/pixl_imaging/_orthanc.py

```diff
@@ -148,11 +181,15 @@ async def job_state(self, job_id: str) -> Any:
     async def _get(self, path: str) -> Any:
         async with (
             aiohttp.ClientSession() as session,
-            session.get(f"{self._url}{path}", auth=self._auth, timeout=10) as response,
+            session.get(
+                f"{self._url}{path}",
+                auth=self._auth,
+                timeout=self.http_timeout,
+            ) as response,
         ):
             return await _deserialise(response)
 
-    async def _post(self, path: str, data: dict, timeout: Optional[float] = None) -> Any:
+    async def _post(self, path: str, data: dict, timeout: int) -> Any:
```

Comment:
I wonder if all posts should just use `self.http_timeout` and not take this as an argument?

Reply:
ah yeah that makes more sense